web/src/components/task/task-sidebar/finish-button/utils.ts

```diff
@@ -1,7 +1,29 @@
+const LISTED_PAGES_LIMIT = 10;
+const SLICE_START_INDEX = 0;
+const SLICE_END_INDEX = LISTED_PAGES_LIMIT - 1;
+
+const getPagesList = (pages: number[]) => pages?.join(', ');
```

Comment:
Do we really need to use "?" here. Based on "number[]" type, it's always defined

Reply:
Addressed